### ISSUE TITLE
Squiz.Functions.MultiLineFunctionDeclaration.SpaceBeforeOpenBrace allows newlines in between closing parenthesis and opening brace.

### ISSUE THREAD

**Issue by austinderrick** [squizlabs/PHP_CodeSniffer#3845]
## Describe the bug
Squiz.Functions.MultiLineFunctionDeclaration.SpaceBeforeOpenBrace allows newlines in between closing parenthesis and opening brace. I believe this rule should require the closing parenthesis and opening brace to be on the same line.

### Code sample
Expected to Pass:
```php
    public static function hasUsername(
        $managedInstanceURL,
        $apiKey,
        $username,
        $useCache = true
    ) {
```

Passes, but shouldn't:
```php
    public static function hasUsername(
        $managedInstanceURL,
        $apiKey,
        $username,
        $useCache = true
    )
    {
```

## Versions (please complete the following information)

| | |
|-|-|
| Operating System | MacOS 13.4
| PHP version | 8.0
| PHP_CodeSniffer version 3.7.2 (stable) by Squiz (http://www.squiz.net)
| Standard | Squiz
| Install type | PHAR

## Please confirm:

- [x] I have searched the issue list and am not opening a duplicate issue.
- [x] I confirm that this bug is a bug in PHP_CodeSniffer and not in one of the external standards.
- [x] I have verified the issue still exists in the `master` branch of PHP_CodeSniffer.

**Comment by jrfnl** [comment]
@austinderrick I cannot reproduce your issue.

When I run the sniff against your example code, I get the following error, which sounds to me exactly what you are looking for:
```
 | ERROR | [x] The closing parenthesis and the opening brace of a multi-line function declaration must be on the same line
 |       |     (Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace)
```

I suspect the above error code may be excluded from within your custom ruleset ?
